### ISSUE TITLE
ComBat parametric adjustment returning NANs in Matlab 

### ISSUE THREAD

**Issue by leotozzi88** [Jfortin1/ComBatHarmonization#31]
Dear Experts, 

I am running ComBat in Matlab version 9.7.0.1190202 (R2019b). 
When I run the batch correction using parametric adjustments, I get all NANs. This has been a recurring issue for me, even in other datasets. 
There are no NANs in my data and, in the past, the non-parametric adjustment has worked.
Any idea what might be causing this? 
Thank you, 

Leonardo Tozzi

**Comment by Jfortin1** [comment]
@leotozzi88 Hi Leonardo, are there features that have a constant values across images in your data?

**Comment by leotozzi88** [comment]
Indeed, I think this was probably the issue: I tried running the analysis in R and I got the warning saying that there were constant features. After removing those, the analysis ran.

**Comment by Jfortin1** [comment]
@leotozzi88 Note to myself: add warning message for the Matlab version for such scenarios
